data_steward/analytics/cdr_ops/retraction_validation.py

```diff
@@ -144,10 +144,8 @@
 RIGHT JOIN pids p2
 on fr.fact_id_2 = p2.person_id
 where domain_concept_id_1 = 56
-''').render(project=project_id, dataset=q4_dataset)
-retraction_status_query = (f'{rids_query}\n{fact_relationship_retraction}'
-                           if is_deidentified == 'true' else
-                           f'{pids_query}\n{fact_relationship_retraction}')
+''').render(project=project_id, dataset=new_dataset)
+retraction_status_query = f'{pids_query}\n{fact_relationship_retraction}'
```

Comment:
so, we don't need the `is_deidentified` check in the `retraction_status_query` anymore?